### PR TITLE
Added CVE-2019-18571,18572,18573

### PR DESCRIPTION
**Pull request by DellEMCProductSecurity**

### CODE REVIEW

File: 2019/18xxx/CVE-2019-18752.json [hunk 1]
```diff
@@ -1,17 +1,70 @@
 {
```
Comment (cve-team) [inline 2019/18xxx/CVE-2019-18752.json]: CNA(s) of user (dell) do not match owning CNA.

Comment (DellEMCProductSecurity) [reply 2019/18xxx/CVE-2019-18752.json]: Not sure what happened here. We received this block of CVEs on 10/29/2019 after requesting more allocation on 10/28/2019. CVE-2019-18752 was among them.

Comment (EvansJonathan) [reply 2019/18xxx/CVE-2019-18752.json]: Looks like you have a typo in the CVE ID.  CVE-2019-18**57**2, but you submitted CVE-2019-18**75**2.

Comment (DellEMCProductSecurity) [reply 2019/18xxx/CVE-2019-18752.json]: Ah oops, Ok I think we may be better off canceling this pull request and I'll reopen one with the correction.

**Pull request closed without merging**